crates/graphql/src/schema.rs

```diff
@@ -690,6 +693,75 @@ impl<'a> From<models::AuctionHouse<'a>> for AuctionHouse {
     }
 }
 
+#[derive(Debug, Clone)]
+struct PublicBid {
+    pub address: String,
+    pub trade_state: String,
+    pub wallet: String,
+    pub token_mint: String,
+    pub price: Lamports,
+    pub trade_state_bump: i32,
+    pub activated_at: Option<DateTime<Utc>>,
```

Comment:
I can change this to NaiveDateTime but if we want date and time then this should be okay